recipes/flex/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class FlexConan(ConanFile):
+    name = "flex"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/westes/flex"
+    description = "Flex, the fast lexical analyzer generator"
+    topics = ("conan", "flex", "lex", "lexer", "lexical analyzer generator")
+    license = "BSD-2-Clause"    
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    requires = ("m4/1.4.18",)
```

Comment:
Should `m4` be a `build_requires`?

Reply:
it looks like it should not: https://github.com/conan-io/conan-center-index/pull/1195#issuecomment-623687788